OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/VideoDownloadQualityDialogFragment.kt

```diff
@@ -55,9 +55,8 @@ class VideoDownloadQualityDialogFragment(
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        environment.analyticsRegistry.trackScreenView(
-            Analytics.Screens.VIDEO_DOWNLOAD_QUALITY, null,
-            Analytics.Values.SCREEN_NAVIGATION
+        environment.analyticsRegistry.trackEvent(
```

Comment:
Please revisit its logic as this method is passing `Analytics.Values.SCREEN_NAVIGATION` as courseId.

Reply:
This is the underlying function being called: 

```
    @Override
    public void trackEvent(@NonNull String eventName, @NonNull String biValue) {
        final SegmentEvent aEvent = new SegmentEvent();
        aEvent.properties.putValue(Keys.NAME, biValue);
        trackSegmentEvent(eventName, aEvent.properties);
    }

No need to update it. 👍🏻

Reply:
Updating it to `trackScreenView`.